engine/getstarted-voting-app/deploy-app.md

```diff
@@ -11,7 +11,19 @@ deploy the voting application to the swarm you just created.
 
 The `docker-stack.yml` file must be located on a manager for the swarm where you want to deploy the application stack.
 
-1.  Get `docker-stack.yml` either from the lab or by copying it from the example given here.
+1.  Get [`docker-stack.yml`](https://github.com/docker/example-voting-app/blob/master/docker-stack.yml) either from source code in the lab or by copying it from the example given [here](https://docs.docker.com/engine/getstarted-voting-app/#/docker-stackyml-deployment-configuration-file).
+
+    If you prefer to download the file directly from our GitHub
+    repository rather than copy it from the documentation, you can use a tool like `curl`. This `curl` command downloads the raw file to the current directory on your local host:
+
+    ```
+     curl -o docker-stack.yml https://raw.githubusercontent.com/docker/example-voting-app/master/docker-stack.yml
+    ```
+
+    >**Tips:**
```

Comment:
I needed to because I have two separate notes I want to tell them, and they are rather unrelated, so they are bullets under the **Tips** label. Similar to the[ important notes](https://docs.docker.com/docker-for-mac/) we needed to call out in Docker for Mac